generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-update.component.ts.ejs

```diff
@@ -201,17 +210,4 @@ export class <%= entityAngularName %>UpdateComponent implements OnInit {
         return option;
     }
     <%_ } _%>
-    get <%= entityInstance %>() {
-        return this._<%= entityInstance %>;
-    }
-
-    set <%= entityInstance %>(<%= entityInstance %>: I<%= entityAngularName %>) {
```

Comment:
how is this related to e2e improvements, and if the moment stuff is not done in setter does it work for all cases?

Reply:
ok